ext/IntervalArithmeticsIntervalSetsExt.jl

```diff
@@ -0,0 +1,27 @@
+module IntervalArithmeticsIntervalSetsExt
+
+import IntervalSets as IS
+import IntervalArithmetic as IA
+
+
+# BareInterval <- IS.Interval
+Base.convert(::Type{IA.BareInterval}, i::IS.Interval) = IA.bareinterval(IS.endpoints(i)...)
+IA.bareinterval(i::IS.Interval) = IA.bareinterval(IS.endpoints(i)...)
+
+# BareInterval -> IS.Interval
+Base.convert(::Type{IS.Interval}, i::IA.BareInterval) = IS.Interval(IA.inf(i), IA.sup(i))
+IS.Interval(i::IA.BareInterval) = IS.Interval(IA.inf(i), IA.sup(i))
+
+# Interval <- IS.Interval
+Base.convert(::Type{IA.Interval}, i::IS.Interval{<:Any,<:Any,T}) where {T} = convert(IA.Interval{float(T)}, i)
```

Comment:
Looking at this line, it seems that `IS.Interval` can have bounds that are of different real types (eg `Rational`, or even maybe `Int`)?
In our case, `IA.Interval` can have bounds that are either `Rational` or `AbstractFloat`.
Hence, if you would like to preserve as much as possible the same bound type as the initial interval, I suggest the following:
```Julia
Base.convert(::Type{IA.Interval}, I::IS.Interval{<:Any,<:Any,T}) where {T} = convert(IA.Interval{IA.promote_numtype(T, T)}, i)
```